src/components/home/index.js

```diff
@@ -84,7 +83,10 @@ class Home extends React.Component {
       <DocumentTitle title={this.title()}>
         {
           <div>
-            {this.props.selectedOrganization ? (
+            <EmptyStateDisplay
```

Comment:
I'm not sure about this EmptyStateDisplay component refactoring. 

It seems to run all the code in the child component whereas the ternary would skip executing it.